README.md

```diff
@@ -69,3 +69,15 @@ $ forge --help
 $ anvil --help
 $ cast --help
 ```
+
+## Validate Upgrade Safety 
+1. Checkout to the deployed implementation commit 
+2. Copy the YieldDistributor to test/upgrades/<version>/YieldDistributor.sol
```

Comment:
Copy `YieldDistributor.sol` to "test/upgrades/<version>/YieldDistributor.sol"

Reply:
addressed with 147bbb8